src/test/java/io/confluent/examples/streams/IntegrationTestUtils.java

```diff
@@ -277,4 +279,87 @@ private static boolean continueConsuming(final int messagesConsumed, final int m
         30000,
         "Expected values not found in WindowStore"); }
 
+  static <K, V> Map<K, V> drainTableOutput(final String outputTopic,
+                                           final TopologyTestDriver topologyTestDriver,
+                                           final Deserializer<K> keyDeserializer,
+                                           final Deserializer<V> valueDeserializer) {
+
+    final Map<K, V> results = new LinkedHashMap<>();
+    while (true) {
+      final ProducerRecord<K, V> record = topologyTestDriver.readOutput(outputTopic, keyDeserializer, valueDeserializer);
+      if (record == null) {
+        break;
+      } else {
+        results.put(record.key(), record.value());
+      }
+    }
+    return results;
+  }
+
+  static <K, V> void produceKeyValuesSynchronously(final String inputTopic,
```

Comment:
Do we need to add `Synchronously` to the name? `TopologyTestDriver` does not support any async stuff anyway.

Reply:
Not sure; I left the name so it would clearly be a "peer" to the other `produceKeyValuesSynchronously`. I.e., functionally equivalent, but for TopologyTestDriver instead.

I figured this would help code readability also, so it would continue to be self-documenting for people (esp those not familiar with the TopologyTestDriver) reading the tests.

Reply:
Ack. Fine with me. Was just a thought.